ui/app/components/auth-jwt.js

```diff
@@ -145,12 +146,8 @@ export default Component.extend({
       }
     }
 
-    // defer closing of the window, but continue executing the task
-    later(() => {
-      this.closeWindow(oidcWindow);
-    }, WAIT_TIME);
```

Comment:
I'm just wondering why this pause was needed in the first place and if there might be a case where removing it will cause a regression?

Reply:
Also, if we don't need it then the `WAIT_TIME` variable can also be removed.

Reply:
The WAIT_TIME variable is used other places 👍  

My suspicion is that we wanted to close the window no matter the outcome of the next request, so wrapping this in a later loop prevented duplicative code. I tested that the window closes if the adapter method errors out and tried to handle all cases.

Reply:
Gotcha thanks!